vars/main-srsran.yml

```diff
@@ -0,0 +1,88 @@
+k8s:
+  rke2:
+    version: v1.23.4+rke2r1
+    config:
+      token: purdue-k8s-rke2
+      port: 9345
+      params_file:
+        master: "deps/k8s/roles/rke2/templates/master-config.yaml"
+        worker: "deps/k8s/roles/rke2/templates/worker-config.yaml"
+
+  kubectl:
+    version: v1.23.15
+
+  helm:
+    version: v3.10.3
+
+core:
+  standalone: true		# set to false to place under control of ROC
+  data_iface: ens18
+  values_file: "deps/5gc/roles/core/templates/sdcore-5g-values.yaml"
+  ran_subnet: "172.20.0.0/16"	# set to empty string to get subnet from 'data_iface'
+  helm:
+    local_charts: false		# set chart_ref to local path name if true
+    chart_ref: aether/sd-core
+    chart_version: 1.1.0
+  upf:
+    ip_prefix: "192.168.252.0/24"
+    iface: "access"
+    mode: af_packet		# Options: af_packet or dpdk
+    default_upf:
+      ip:
+        access: "192.168.252.3/24"
+        core:   "192.168.250.3/24"
+      ue_ip_pool: "172.250.0.0/16"
+  amf:
+    ip: "10.76.28.113"
+
+srsran:
+  docker:
+    container:
+      gnb_image: aetherproject/srsran-gnb:0.0.1-dev
+      ue_image: aetherproject/srsran-ue:0.0.1-dev
+    network:
+      data_iface: ens18
+      name: host
+      subnet: "172.20.0.0/16"
+      bridge:
+        name: rfsim5g-public
+  simulation: true
+  gnb:
+    conf_file: deps/srsran/roles/gNb/templates/gnb_zmq.conf
+    ip: "172.20.0.2"
+  ue:
+    conf_file: deps/srsran/roles/uEsimulator/templates/ue_zmq.conf
```

Comment:
Are you using all parameters/variables set here?

Reply:
Yes, except data_iface/bridge.name all other parameters are used now. I am working on using separate n/w for deploying  srsran (like oai). For that purpose i kept these two parameters above.